app/controllers/invoice_items_controller.rb

```diff
@@ -0,0 +1,22 @@
+class InvoiceItemsController < ApplicationController
+  def update
+    @invoice_item = InvoiceItem.where(item_id: params[:item_id], invoice_id: params[:invoice_id]).first
+    if params[:status]
+      @invoice_item.update(invoice_item_params)
+      redirect_to merchant_invoice_path(params[:merchant_id], params[:invoice_id])
+    end
+  end
+
+  private
+
+  def invoice_item_params
+    if params[:status] == "Pending"
+      params[:status] = 0
+    elsif params[:status] == "Packaged"
+      params[:status] = 1
+    else # == "Shipped"
```

Comment:
could you make this an elsif instead of an else here? just to make sure that it will do exactly what we want it to do.

Reply:
I will follow up with chris on this one

Reply:
I followed up with chris and he agreed that elsif is preferred in this instance because there isn't an alternative route. So I have also changed this one!